test/topology_experimental_raft/test_fencing.py

```diff
@@ -63,3 +99,78 @@ async def test_fence_writes(request, manager: ManagerClient):
         await manager.cql.run_async("update t2 set counter_c=counter_c+1 where pk=1", host=host2)
 
     random_tables.drop_all()
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
```

Comment:
Nice. Suggestion for a follow-up: replace all uses of `skip_in_*` in suite.yaml files with this, which is much better.

BTW. tests that are enabling injections through the REST API already have a `pytest.skip` in them, but this doesn't work for `error_injections_at_startup` obviously. Maybe we should find a way to somehow detect that `error_injections_at_startup` is used and automatically skip the test... but I guess it would be some PITA. The `skip_mode` marker also works.